@navikt/core/react/src/tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,321 @@
+import cl from "classnames";
+import React, {
+  cloneElement,
+  forwardRef,
+  HTMLAttributes,
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from "react";
+import { composeEventHandlers, Detail, useEventListener } from "..";
+import {
+  useFloating,
+  arrow as flArrow,
+  shift,
+  autoUpdate,
+  flip,
+  hide,
+} from "@floating-ui/react-dom";
+import mergeRefs from "react-merge-refs";
+import Portal from "./portal";
+import { useId } from "../util";
+
+export interface TooltipProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * Element tooltip anchors to
+   */
+  children: React.ReactElement & React.RefAttributes<HTMLElement>;
```

Comment:
Er det noen grunn til å ikke bare bruke `ReactNode` her?
```suggestion
  children: React.ReactNode;
```

Reply:
Ja er ikke mulig å bruke flere children, eks 
```jsx
<Tooltip>
    <Child1/>
    <Child2/>
</Tooltip>
```

Er heller ikke lov med react-fragments
```jsx
<Tooltip>
    <>
        <Child1/>
    </>
</Tooltip>
```